src/helpers.ts

```diff
@@ -290,6 +290,9 @@ export function hasFeature(contactReceiver: threema.ContactReceiver,
             log.warn(`Contact receiver with id ${contactReceiver.id} has featureMask 0`);
             return false;
         }
+        if (feature === threema.ContactReceiverFeature.NONE) {
+            return true;
+        }
```

Comment:
:confused:

Reply:
Slightly awkward but not any more awkward than the `undefined` check above. :smirk_cat: